spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/ChannelBindingServiceProperties.java

```diff
@@ -174,4 +193,11 @@ public Properties getProducerProperties(String outputChannelName) {
 		}
 	}
 
+	public String getBinder(String channelName) {
```

Comment:
I don't particularly like the methods, but they're an expedient way of doing a null-safe property lookup. Maybe we can invest some effort in that direction ...